src/main/java/org/usfirst/frc/team4099/robot/subsystems/Arm.kt

```diff
@@ -0,0 +1,163 @@
+package org.usfirst.frc.team4099.robot.subsystems
+import org.usfirst.frc.team4099.robot.Constants
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard
+import org.usfirst.frc.team4099.robot.loops.Loop
+import edu.wpi.first.wpilibj.DoubleSolenoid
+import com.ctre.phoenix.motorcontrol.ControlMode
+import com.ctre.phoenix.motorcontrol.FeedbackDevice
+import com.ctre.phoenix.motorcontrol.can.TalonSRX
+
+/**
+ * @author Team 4099
+ *
+ * This class is the constructor for the Arm subsystem
+ *
+ * @constructor Creates the Arm subsystem
+ *
+ */
+class Arm private constructor() : Subsystem {
+
+    private val masterSRX = TalonSRX(Constants.Arm.MASTER_SRX_ID)
+    private val slaveSRX1 = TalonSRX(Constants.Arm.SLAVE_SRX_1_ID)
+    private val slaveSRX2 = TalonSRX(Constants.Arm.SLAVE_SRX_2_ID)
+    private val slaveSRX3 = TalonSRX(Constants.Arm.SLAVE_SRX_3_ID)
+
+    private val brake = DoubleSolenoid(Constants.Arm.BRAKE_FORWARD_CHANNEL, Constants.Arm.BRAKE_REVERSE_CHANNEL)
+
+
+    var movementState = MovementState.STATIONARY
+    private var armPower = 0.0
+
+    var armState = ArmState.EXCHANGE
+    var targetPos: Double = 0.0
+
+    var useVelocityControl : Boolean = false
+
+    enum class MovementState {
+        UP, STATIONARY, DOWN
+    }
+
+    enum class ArmState {
+        LOW, EXCHANGE, HIGH
+    }
+
```

Comment:
Add some open loop state for `ArmState`. When the arm is controlled manually, there's a chance it won't be in any of the three main states.